File: src/run-interactive.ts
import "dotenv/config";
import { weatherAgent } from "./agents/weather.agent";
import { gemini20FlashModel } from "./core/ai.provider";
import { runInteractive } from "./core/runner.interactive";

async function main() {
  await runInteractive(weatherAgent, gemini20FlashModel);
}

main().catch((error) => {
  console.error("Error:", error);
  process.exit(1);
});

File: run_interactive.py
import asyncio
from core.ai_provider import gemini_2_0_flash_model
from agents.weather_agent import weather_agent
from core.runner_interactive import run_interactive

if __name__ == "__main__":
    asyncio.run(run_interactive(weather_agent, gemini_2_0_flash_model)) 
